pkg/cmd/time-entry/util/report.go

```diff
@@ -91,18 +103,44 @@ func PrintTimeEntry(
 	b := config.GetBool(cmdutil.CONF_SHOW_TOTAL_DURATION)
 	config.SetBool(cmdutil.CONF_SHOW_TOTAL_DURATION, false)
 
-	err := PrintTimeEntries(ts, out, config, of)
+	err := PrintTimeEntries(updateTimeZone(ts, of), out, config, of)
```

Comment:
you do it in the function, so you don't need to do it out here
```suggestion
	err := PrintTimeEntries(ts, out, config, of)
```